### PR TITLE
Calculate votes count and scores even if shutter is enabled

### PR DESCRIPTION
**Pull request by ChaituVR**
Works only with https://github.com/snapshot-labs/snapshot.js/pull/791

### CODE REVIEW

File: src/scores.ts [hunk 1]
```diff
@@ -166,7 +166,18 @@ export async function updateProposalAndVotes(proposalId: string, force = false)
...
 
   // Get results
-  const voting = new snapshot.utils.voting[proposal.type](proposal, votes, proposal.strategies);
+  const encryptedChoice = proposal.privacy === 'shutter' && proposal.state !== 'closed';
+
+  const voting = new snapshot.utils.voting[proposal.type](
+    proposal,
+    votes,
+    proposal.strategies,
+    undefined,
```
Comment (samuveth) [inline src/scores.ts:175]: We can remove `undefined` and `{ encryptedChoice }` as they are no longer needed with https://github.com/snapshot-labs/snapshot.js/pull/791

File: src/scores.ts [hunk 2]
```diff
@@ -166,7 +166,18 @@ export async function updateProposalAndVotes(proposalId: string, force = false)
...
+    {
+      encryptedChoice
+    }
+  );
+
```
Comment (samuveth) [inline src/scores.ts:180]:
```suggestion

  const voting = new snapshot.utils.voting[proposal.type](
    proposal,
    votes,
    proposal.strategies
  );

```

File: src/scores.ts [hunk 3]
```diff
@@ -167,6 +167,7 @@ export async function updateProposalAndVotes(proposalId: string, force = false)
 
   // Get results
   const voting = new snapshot.utils.voting[proposal.type](proposal, votes, proposal.strategies);
+
```
Comment (ChaituVR) [inline src/scores.ts:170]:
```suggestion
```

**Review by samuveth — APPROVED** [review]
utACK

**Pull request merged**